addons/docs/src/blocks/Meta.tsx

```diff
@@ -28,9 +28,15 @@ function renderAnchor() {
  * This component is used to declare component metadata in docs
  * and gets transformed into a default export underneath the hood.
  */
-export const Meta: FC<MetaProps> = () => {
-  const params = new URL(document.location).searchParams;
-  const isDocs = params.get('viewMode') === 'docs';
+export const Meta: FC<MetaProps> = ({ of }) => {
+  let isDocs = true;
+  if (document) {
```

Comment:
When is there no document?

Reply:
In the nextra demo `global.document` is not defined. I guess because it SSRs. It is best if we write our components to not make any assumptions about the rendering environment to be as portable as possible.